cmd/notation/verify.go

```diff
@@ -157,6 +159,24 @@ func checkVerificationFailure(outcomes []*notation.VerificationOutcome, printOut
 	// write out on failure
 	if err != nil || len(outcomes) == 0 {
 		if err != nil {
+			var errTrustStore truststore.TrustStoreError
+			if errors.As(err, &errTrustStore) {
+				if errors.Is(err, fs.ErrNotExist) {
+					return fmt.Errorf("%w. Use command 'notation cert add' to create and add trusted certificates to the trust store", errTrustStore)
```

Comment:
Is this error for missing trust store or trust certificate?

Reply:
This is for missing trust store, but the next actionable step for user is still `notation cert add`. This is because `notation cert add` adds the certificates, if the trust store does not exist, it will create the trust store in this process.